modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java

```diff
@@ -574,17 +584,18 @@ public void checkCPInstancesByDisplayDate(long cpDefinitionId)
 			serviceContext.setWorkflowAction(WorkflowConstants.ACTION_PUBLISH);
 
 			CPDefinition cpDefinition =
-				cpDefinitionLocalService.getCPDefinition(cpDefinitionId);
+				cpDefinitionLocalService.getCPDefinition(
```

Comment:
I don't get this, can you explain me why we need this?

Reply:
Now that I reviewed with @alessiorendina I got it. I think this change is right, thanks!